packages/api-v2/src/AuthProvider.tsx

```diff
@@ -184,25 +184,31 @@ const AuthProvider = ({ loginIDKey, children, cookieTimeout, selectDefaultAccoun
                 setIsAuthorized(false);
                 await mutateAsync({ payload: { authorize: token || '' } })
                     .then(res => {
-                        setIsAuthorized(true);
-                        processAuthorizeResponse(res);
-                        setIsLoading(false);
-                        setIsInitializing(false);
-                        setIsSuccess(true);
-                        setLoginid(res?.authorize?.loginid ?? '');
+                        if (isMounted) {
+                            setIsAuthorized(true);
+                            processAuthorizeResponse(res);
+                            setIsLoading(false);
+                            setIsInitializing(false);
+                            setIsSuccess(true);
+                            setLoginid(res?.authorize?.loginid ?? '');
+                        }
```

Comment:
Added isMounted to AuthProvider initialize function as well due to memory leak console error which intermittently popups from DerivAppsGetAccount